docs/developer/style.md

```diff
@@ -0,0 +1,215 @@
+# Go Style Guide in Vald
+
+## Table of Contents
+
+- [Introduction](#Introduction)
+- [Style](#Style)
+- [Test](#Test)
+	- [Table-Driven-Test](#Table-Driven-Test)
+
+## Introduction
+
+## Style
+
+## Test
+
+This section will describe the rule for writing test code in Vald.
+
```

Comment:
```suggestion
Test guideline has the 2 important rules for continuous quality.
1. Use Table-Driven-Test
2. Test all possible patterns
```
@kevindiu Is it no problem with grammar?

Reply:
I think it is better.
```suggestion
Testing guideline has 2 important rules for the coding quality and readability.
```

Reply:
I'd like to write `Test all possible patterns`.
Where should this sentence be written?

Reply:
You mean the testing coverage?

Reply:
I mean all user possible cases